packages/angular-material/src/other/master-detail/master.ts

```diff
@@ -0,0 +1,98 @@
+import { Component } from '@angular/core';
+import * as _ from 'lodash';
+import { JsonFormsControl } from '@jsonforms/angular';
+import { map } from 'rxjs/operators/map';
+import { NgRedux } from '@angular-redux/store';
+import {
+    ControlElement,
+    Generate,
+    JsonFormsState,
+    RankedTester,
+    rankWith,
+    resolveSchema,
+    toDataPath,
+    uiTypeIs
+} from '@jsonforms/core';
+import { MasterDetailService } from './master-detail.service';
+
+const keywords = ['#', 'properties', 'items'];
+
+export const removeSchemaKeywords = (path: string) => {
+    return path.split('/').filter(s => !_.some(keywords, key => key === s)).join('.');
+};
+
+@Component({
+    selector: 'jsonforms-list-with-detail-master',
+    template: `
+    <div fxLayout='row' fxLayoutGap='16px'>
+        <div flFlex>
```

Comment:
I would suggest to use https://v5.material.angular.io/components/sidenav/overview instead of plain divs
Even when using plain divs, the full width of the screen should be used imho